### ISSUE TITLE
Use a more neutral name for the default branch instead of master when initializing a new repository

### ISSUE THREAD

**Issue by sleeve** [go-gitea/gitea#11886]
There has recently been a lot of discussion on improving technical jargon and terminology to remove more racially insensitive terms like "master/slave" or "whitelist/blacklist" to make tech even more inclusive.

https://www.cnet.com/news/master-and-slave-tech-terms-face-scrutiny-amid-anti-racism-efforts/
https://boleary.dev/blog/2020-06-10-i-was-wrong-about-git-master.html
https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/
https://gitlab.com/gitlab-org/gitlab/-/issues/221164
https://gitlab.com/gitlab-org/gitlab/-/issues/220906
https://lore.kernel.org/git/20200609223624.GO6569@camp.crustytoothpaste.net/T/#mf6bab91e35ad94fe6ce272615219f887b9b8b440
https://petapixel.com/2020/06/06/its-time-to-end-the-terms-slave-and-master-in-photography/
https://www.hanselman.com/blog/EasilyRenameYourGitDefaultBranchFromMasterToMain.aspx
https://twitter.com/leahculver/status/1269109776983547904
https://www.notion.so/9560a2fdb9364c1aa9e1ede6e6e84225?v=097d37e5e18d452b81af5712df427a56

I think it would be a great addition to gitea to change the default branch name from `master` to something more neutral like `main` when initializing a new repository. Or at least offer the option of setting a different default name somewhere within Settings.

Thanks!

**Comment by techknowlogick** [comment]
Thank you for this ticket. This is something Gitea already has implemented for a long time.

**Comment by sleeve** [comment]
@techknowlogick 

Apologies for the duplicate but I did not see it when searching the issues. After doing a more in-depth search I was able to find this issue and subsequent pull request for it.

https://github.com/go-gitea/gitea/issues/9542
https://github.com/go-gitea/gitea/pull/10803

Hopefully by linking these it helps others not make the same mistake as I did. I think its also fair to note that this isn't actually in the latest 1.11.6 stable release so I hope you see how that can be confusing for users. It looks like it went in just a few months ago and is in the 1.12.0 master branch 🤦‍♂️ which hasn't quite been finalized.
